iOS/G3MiOSSDK/Commons/Rendererers/PlanetRenderer.cpp

```diff
@@ -32,24 +32,29 @@
 #include "Layer.hpp"
 #include <algorithm>
 
+#include "MaxLevelTileLODTester.hpp"
+#include "ProjectedCornersDistanceTileLODTester.hpp"
+#include "TimedTileLODTester.hpp"
+#include "MaxFrameTimeTileLODTester.hpp"
+
```

Comment:
PlanetRenderer creates lazily a default implementation of the TileLODTester in case none is porvided. This implementation needs the LayerTileRenderParameters, known by the PlanetRenderer.